lib/kvdb/kvdb_rparams.c

```diff
@@ -885,6 +885,21 @@ static const struct param_spec pspecs[] = {
             },
         },
     },
+    {
+        .ps_name = "csched_full_compact",
```

Comment:
Is there any way to do this without the added rparam? This almost seems like more of a command to me.

Reply:
Having it be a REST param enables making full compaction an online option at some point.